main/realmain.cc

```diff
@@ -591,8 +591,7 @@ int realmain(int argc, char *argv[]) {
             auto foundHashes = nullptr;
             auto canceled = pipeline::name(*gs, absl::Span<ast::ParsedFile>(indexed), opts, *workers, foundHashes);
             ENFORCE(!canceled, "There's no cancellation in batch mode");
-            pipeline::buildPackageDB(*gs, absl::MakeSpan(indexed), opts, *workers);
-            pipeline::setPackageForSourceFiles(*gs, inputFilesSpan, opts);
+            pipeline::buildPackageDB(*gs, absl::MakeSpan(indexed), inputFilesSpan, opts, *workers);
```

Comment:
I think it is actually great to have fewer, top-level "public" APIs exposed by packager.

Of course, the eventual goal is that we get back to a world where there is only `pipeline::name` and `pipeline::resolve`, i.e., there isn't even a `buildPackageDB` step. This is how we will get incrementality "for free," because namer and resolver already support incrementality.